index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="KO">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <nav class="navigation-bar">
+        <a href="/"><img src="./image/logo.png" alt="linkbrary logo" width="133px" height="24px"></a>
+        <span class="login-button"><a href="/signin.html">로그인</a></span>
+    </nav>
+    <div class="main">
```

Comment:
div태그 대신에 main 태그를 사용해주시면 더 좋을 것 같아요~

```suggestion
    <main class="main">
```